AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -52,6 +52,8 @@ disk.yandex.*##.mail360-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/78285
 nu.nl##.zone > div[id^="block-"][data-context*="Adverteerders"]
 nu.nl##.zone > div[id^="block-"][data-context*="Aangeboden door NUshop"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/78792
+pinnacle.com##a[data-gtm-id^="promo_banner_"][rel^="noopener"]
```

Comment:
Do you need `[rel^="noopener"]`? `pinnacle.com##a[data-gtm-id^="promo_banner_"]` works ok on my end.

Reply:
Yeah, it works too. Just to narrow it down.

Reply:
According to the code, without `[rel^="noopener"]` should be ok.

Reply:
Okay. I will remove it.

Reply:
Okay. I will remove it.